packages/avatar/contracts/nft-collection/ERC721BurnUpgradeable.sol

```diff
@@ -11,18 +11,8 @@ import {ERC721Upgradeable} from "@openzeppelin/contracts-upgradeable-5.0.2/token
  * @notice Baseline ERC721 contract to be used by the NFTCollection contract
  * @dev provides the "burn memory" functionality: keeping track of who burned what token
  */
-abstract contract ERC721BurnMemoryUpgradeable is ERC721Upgradeable {
+abstract contract ERC721BurnUpgradeable is ERC721Upgradeable {
```

Comment:
Does it make sense to keep this as a separate contract now that it has been slimmed down so much?

Reply:
You are right, just removed the base clase